index.html

```diff
@@ -65,6 +65,24 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="why">
+            <h2 class="why-h2">Varför?</h2>
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee1.png" alt="drawn picture of a bee from the top">
```

Comment:
I believe the `alt` text could be in Swedish, as the other website copy is in Swedish :)